src/storage-types/src/sinks.proto

```diff
@@ -36,6 +36,14 @@ message ProtoStorageSinkDesc {
     mz_repr.antichain.ProtoU64Antichain as_of = 11;
     bool with_snapshot = 12;
     uint64 version = 13;
+    ProtoSinkPartitionStrategy partition_strategy = 14;
+}
+
+message ProtoSinkPartitionStrategy {
+    oneof kind {
+        google.protobuf.Empty v0 = 1;
+        google.protobuf.Empty v1 = 2;
+    }
```

Comment:
nit: unless we are planning to add additional sub-options to `partition strategy` this could just be a normal proto enum rather than a oneof, right?

Reply:
Oh, I didn't know proto enums where a thing! This is an ephemeral message that will get deleted once everyone migrates to v1 so I'm leaning towards leaving it as-is. I'll definitely have this in mind next time I create a similarly shaped proto message though!